arith_amd64.s

```diff
@@ -0,0 +1,85 @@
+//go:build amd64 && !purego
+
+#include "textflag.h"
+
+// func andAVX2(z, x, y *[32]byte)
+TEXT ·andAVX2(SB), NOSPLIT, $0-24
+    MOVQ z+0(FP), AX
+    MOVQ x+8(FP), BX
+    MOVQ y+16(FP), CX
+
+    VMOVDQU (BX), Y0
+    VMOVDQU (CX), Y1
+
+    VPAND Y0, Y1, Y2    // Y2 = Y0 & Y1
+
+    VMOVDQU Y2, (AX)
+
+    VZEROUPPER
+    RET
+
+// func orAVX2(z, x, y *[32]byte)
+TEXT ·orAVX2(SB), NOSPLIT, $0-24
+    MOVQ z+0(FP), AX
+    MOVQ x+8(FP), BX
+    MOVQ y+16(FP), CX
+
+    VMOVDQU (BX), Y0
+    VMOVDQU (CX), Y1
+    VPOR Y0, Y1, Y2
+    VMOVDQU Y2, (AX)
+    VZEROUPPER
+    RET
```

Comment:
Please use the same "form" for these three methods (and or xor). For example: the first 5 instructions without spacing, then the VPAND or VPOR or VPXOR stand-alone, and then the tail.  

```suggestion
    MOVQ z+0(FP), AX
    MOVQ x+8(FP), BX
    MOVQ y+16(FP), CX
    VMOVDQU (BX), Y0
    VMOVDQU (CX), Y1
    
    VPOR Y0, Y1, Y2
    
    VMOVDQU Y2, (AX)
    VZEROUPPER
    RET
```